test/event_test.dart

```diff
@@ -124,7 +124,8 @@ void main() {
           ],
           [
             "p",
-            "052acd328f1c1d48e86fff3e34ada4bfc60578116f4f68f296602530529656a2"
+            "052acd328f1c1d48e86fff3e34ada4bfc60578116f4f68f296602530529656a2",
```

Comment:
I fix up this change which is not necessary

Reply:
I tried to fixup your commit, but not sur it worked…

Can you fixup and push this commit without changes on `test_event.dart`

Reply:
done 👍